<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(63, 131, 241);
background-color: rgb(165, 160, 159);
alternate-background-color: rgb(161, 161, 161);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="TelaCompras">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>701</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="SistemaComprasLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SISTEMA DE COMPRAS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>420</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Total: R$</string>
    </property>
   </widget>
   <widget class="QLabel" name="TotalLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>420</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="CodigoLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>470</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CÃ³digo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Codigo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoAdd">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>480</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>Adicionar</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>701</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoRem">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>480</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);
background-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Remover</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoFin">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>480</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);
background-color: rgb(255, 228, 21);
background-color: rgb(255, 0, 0);
background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Finalizar</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="btnIniciarConsulta">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Iniciar Consulta</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
